plugins/org.eclipse.elk.alg.radial/src/org/eclipse/elk/alg/radial/intermediate/CalculateGraphSize.java

```diff
@@ -59,8 +59,12 @@ public void process(final ElkNode graph, final IElkProgressMonitor progressMonit
         // set up the graph
         double width = maxXPos - minXPos + padding.getHorizontal();
         double height = maxYPos - minYPos + padding.getVertical();
-        graph.setWidth(width);
-        graph.setHeight(height);
+        if (!graph.getProperty(CoreOptions.NODE_SIZE_FIXED_GRAPH_SIZE)) {
+            graph.setWidth(width);
+            graph.setHeight(height);
+        }
+        graph.setProperty(CoreOptions.TOPDOWN_CHILD_AREA_WIDTH, width);
```

Comment:
Why does the algorithm always set these properties and not only in the top-down case?

Reply:
here I think there are two options, either I change the name of the property, because it is not really topdown specific and it could in theory be useful for other stuff in the future or I add an check and only set it in the top-down case. I'm open to both so what do you suggest?

Reply:
If it is not top-down specific you should rename it.